static/js/common/upload-addon.js

```diff
@@ -474,6 +474,12 @@
             $('.binary-source').hide();
             $('.compatible-apps').hide();
           } else {
+            // Pre-check Android or not depending on what we detected in the
+            // manifest.
+            $('#id_compatible_apps .android input').prop(
+              'checked',
+              results.explicitly_compatible_with_android,
```

Comment:
Can we uncheck this checkbox?

Reply:
Ugh I forgot that `readonly` doesn't apply to checkboxes. I'm not sure `disabled` would be the right fix, it needs more testing. I's not blocking this PR since the checkbox will be completely ignored for, it's purely to be nice to the developer.

Reply:
81910a335def5a2449a22bf0541db2798238fac7